src/main/kotlin/io/snyk/plugin/ui/toolwindow/panels/JCEFDescriptionPanel.kt

```diff
@@ -382,6 +421,17 @@ class SuggestionDescriptionPanelFromLS(
                 showAIFixes(fixes);
               };
 
+              window.receiveIgnoreInFileResponse = function (success){
+                console.log('[[receiveIgnoreInFileResponse]]', success);
+                if(success){
+                    ignoreInFileBtn.disabled = true;
+                    console.log('Ignored in file', success);
+                    document.getElementById('ignore-file-issue').disabled = true;
+                }else{
+                    toggleElement(ignoreInFileBtn, "show");
+                    console.error('Failed to apply fix', success);
+                }
+              }
```

Comment:
Can't the whole added part be pushed down to LS?

Reply:
It can, it just depends how we want to use this logic, my question would be : **Should** we push it down to LS? do we plan on using it from LS? eventually with other IDE's than IntelliJ